spec/controllers/api/v1/project_preferences_controller_spec.rb

```diff
@@ -137,4 +137,30 @@
 
     it_behaves_like "is creatable"
   end
+
+  describe "#update_project_settings", :focus do
+    let!(:api_user) { create(:user) }
+    let!(:project) { create(:project, owner: api_user) }
+    let!(:upps) { create(:user_project_preference, user: authorized_user, project: project) }
```

Comment:
don't set the user here as you want to test an authorized project owner updating another user's UPP.